opm/simulators/wells/WellBhpThpCalculator.hpp

```diff
@@ -117,7 +117,25 @@ class WellBhpThpCalculator {
   std::pair<Scalar, Scalar>
   getFloIPR(const WellState<Scalar>& well_state,
             const Well& well, 
-            const SummaryState& summary_state) const;                                                                                              
+            const SummaryState& summary_state) const;
+
+  //! \brief Find limits using brute-force solver.
+  static bool bruteForceBracket(const std::function<Scalar(const Scalar)>& eq,
```

Comment:
any reason to move this function outside of `private`? I think it might be a fallout during the development process.

Reply:
You are right. I placed it back